### ISSUE TITLE
Only monospace fonts are supported

### ISSUE THREAD

**Issue by stefanstranger** [microsoft/vscode#75333]
Due to the fact the following issue is already closed ([https://github.com/microsoft/vscode/issues/51543](https://github.com/microsoft/vscode/issues/51543)) I created a new issue because suggested solution does not work (anymore). 
As soon as I add a new (Nerd) font like this: "terminal.integrated.fontFamily": "LiberationMono NF",

I get a message that only monospace fonts are supported.

- VSCode Version: 1.35.0
- OS Version: 10.0.17763

Steps to Reproduce:

1. Install Nerdfont LiberationMono NF
2. Add the following to your settings.json ("terminal.integrated.fontFamily": "LiberationMono NF")

**Comment by Tyriar** [comment]
You should only be getting that notification if the width of the `i` and `w` characters differ, is the font not actually monospace? 

https://github.com/microsoft/vscode/blob/40bbb20c6006e619e47f29c337ff62a9585b18bd/src/vs/workbench/contrib/terminal/browser/terminalConfigHelper.ts#L58-L67

**Comment by stefanstranger** [comment]
Hi Daniel,

Do you need more information from me?

Stefan [MSFT]

**Comment by Tyriar** [comment]
@stefanstranger is this the font you downloaded? https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/LiberationMono/complete/Literation%20Mono%20Nerd%20Font%20Complete%20Mono%20Windows%20Compatible.ttf

**Comment by stefanstranger** [comment]
@Tyriar I downloaded the following https://github.com/ryanoasis/nerd-fonts/releases/download/v2.0.0/LiberationMono.zip

**Comment by Tyriar** [comment]
Works for me on Windows 10.0.18932.1000:

<img width="535" alt="Screen Shot 2019-07-06 at 10 37 22 AM" src="https://user-images.githubusercontent.com/2193314/60759416-1e353080-9fda-11e9-88bf-06f71df10892.png">

Does Chrome or new Edge pick up the font when you set that as a `font-family` in CSS?

**Comment by Tyriar** [comment]
You could try setting `terminal.integratd.rendererType` to `"dom"` and inspecting the DOM to see what font-family is being set?

**Comment by machsix** [comment]
![Screenshot from 2019-07-12 17-30-54](https://user-images.githubusercontent.com/28209092/61159704-d7a87000-a4ca-11e9-995e-129106e9b9ee.png)

I have the same problem here. The setting for fontfamily of editor text works. But for terminal, it complains the font is not monospace. If I set `terminal.integrated.rendererType` to `dom`, the font-family of DOM element is indeed the name of the font but there is an extra "redenered font" shown at the bottom right panel.

I'm running ArchLinux Linux x64 5.2.0. I installed  `IBM Plex` fonts and `MS` fonts.

VS code version is 1.36.1. Commit number is 2213894e
